doc/nrf/releases/release-notes-latest.rst

```diff
@@ -0,0 +1,31 @@
+.. _ncs_release_notes_latest:
+
+Changes in |NCS| v1.3.99
+########################
+
+The most relevant changes that are present on the master branch of the |NCS|, as compared to the latest release, are tracked in this file.
+Note that this file is a work in progress and might not cover all relevant changes.
+
+
+Change log
+**********
+
+See the `master branch section in the change log`_ for a list of the most important changes.
```

Comment:
Since libraries are tracked separately, and Zephyr is tracked here, I think it is important to make it clear that this changelog only covers the nrf repository somehow. Perhaps something like this:

```suggestion
See the `master branch section in the change log`_ for a list of the most important changes in the ``nrf`` repository.
```

Reply:
It doesn't - the wiki change log also contains entries for bsdlib and the zboss stack, which are in the nrfxlib repo. It's (currently) intended to cover the NCS.
That's why I mentioned it directly under the "Change log" section and did not make a repo subsection for it.

Reply:
OK, how about "most important changes not covered here" or so then?

Reply:
If this is how we plan to use the wiki change log?
My understanding was that the wiki change log is a summary covering all of NCS. And that we need a place ( => this file here) where we can list in more detail what the Zephyr upmerges contain. I would expect though that we mention the upmerge in the wiki as well (since this is where people will probably look first), and then the "not covered here" would be misleading.

Reply:
> My understanding was that the wiki change log is a summary covering all of NCS. And that we need a place ( => this file here) where we can list in more detail what the Zephyr upmerges contain.

Then why is there an nrfxlib entry here too?

That's why I was asking about "not covered here". Since the wiki's changelog seems to cover "everything in NCS that isn't nrfxlib or zephyr", not just "everything in NCS that isn't zephyr".